src/entities/UserSession.ts

```diff
@@ -0,0 +1,84 @@
+import crypto from 'crypto';
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { v4 as uuidv4 } from 'uuid';
+
+import { SerializedUserSession } from '../common/types';
+import User from './User';
+
+@Entity('usersessions')
+class UserSession {
+  /** Universally unique identifier */
+  @PrimaryGeneratedColumn('uuid')
+  uuid: string = uuidv4();
+
+  /** Access token associated with session */
+  @Column('character varying')
+  accessToken = '';
+
+  /** Timestamp of when the session expires (Unix time) */
+  @Column('bigint')
+  expiresAt = '-1';
+
+  /** Refresh token associated with session */
+  @Column('character varying')
```

Comment:
We use `varchar` instead of `character varying` in other entities (they do the same thing). Not a big deal but nice to standardize.